test/e2e/density.go

```diff
@@ -109,25 +97,31 @@ var _ = Describe("Density", func() {
 		// Terminating a namespace (deleting the remaining objects from it - which
 		// generally means events) can affect the current run. Thus we wait for all
 		// terminating namespace to be finally deleted before starting this test.
-		err = deleteTestingNS(c)
+		err = checkTestingNSDeletedExcept(c, ns)
 		expectNoError(err)
 
-		nsForTesting, err := createTestingNS("density", c)
-		ns = nsForTesting.Name
-		expectNoError(err)
 		uuid = string(util.NewUUID())
 
 		expectNoError(resetMetrics(c))
 		expectNoError(os.Mkdir(fmt.Sprintf(testContext.OutputDir+"/%s", uuid), 0777))
 		expectNoError(writePerfData(c, fmt.Sprintf(testContext.OutputDir+"/%s", uuid), "before"))
-		gcloudListNodes()
+
+		Logf("Listing nodes for easy debugging:")
```

Comment:
It's trivial to write if someone will needed it in the future, and I'd rather avoid polluting codebase with 'possibly useful' functions.